### PR TITLE
Added a Press Mirror paragraph

### PR DESCRIPTION
**Pull request by daschuer**
This is a collection of nice articles. 
Hopefully these cross links helps to find these articles and finally the way to us.

### CODE REVIEW

**Comment by Swiftb0y** [comment]
LGTM

File: pages/press.html [hunk 1]
```diff
@@ -63,12 +63,85 @@ <h2>{% trans "Artwork" %}</h2>
...
+
+    <p style="text-indent: 0px;">
+      {% trans "2018-01-17:" %}<br/>
+      <a href="https://www.bonedo.de/artikel/einzelansicht/mixxx-21-das-dj-software-open-source-projekt-legt-nach.html">(DE) bonedo: Mixxx 2.1 - das DJ-Software Open Source Projekt legt nach</a>
+      <p class="pressquote"> "MIXXX ist eine vollständige DJ-Software, für die man keinen Cent berappen muss (aber ruhig etwas spenden sollte) ..." </p>
+    </p>
+
+    <p style="text-indent: 0px;">
+      {% trans "2018-01-15:" %}<br/>
+      <a href="https://djworx.com/open-source-mixxx-gets-v2-1-beta-for-windows-macos-and-linux/">(EN) DJ WORKS: Open source Mixxx gets v2.1 beta for Windows, macOS, and Linux</a>
```
Comment (Pegasus-RPG) [inline pages/press.html]: "(EN) DJWorx"

File: pages/press.html [hunk 2]
```diff
@@ -63,12 +63,85 @@ <h2>{% trans "Artwork" %}</h2>
...
+      <p class="pressquote"> "But in a world of large commercial outfits already extracting money from DJs, we need the little guys doing their own thing. I’m glad that they stuck to their guns, and steadily beaver away at making Mixxx better with every release." </p>
+    </p>
+
+    <p style="text-indent: 0px;">
+      {% trans "2018-10-25:" %}<br/>
+      <a href="https://www.dj-lab.de/ueberblick-die-vier-besten-kostenlosen-dj-programme-2018/">(EN) DJ LAB: Die vier besten kostenlosen DJ-Softwares 2019</a>
```
Comment (Pegasus-RPG) [inline pages/press.html]: This is EN? Looks like DE to me. ;)

**Comment by daschuer** [comment]
Fixed, thank you for review.

**Comment by Pegasus-RPG** [comment]
LGTM. (Is there a way I can easily preview this? Or can you attach a screen shot?)

**Comment by Swiftb0y** [comment]
https://staging.mixxx.org/press/

**Comment by ywwg** [comment]
staging appears to be down but this looks like a good section to add to the site -- thirding the LGTM :)

**Comment by daschuer** [comment]
Ups, I have forget about this.
Hopefully the linkes are mit yet outdated.

**Pull request merged**